src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue

```diff
@@ -197,25 +196,15 @@ export default class TreeBuilderToolbar extends mixins(Routes, Permissions, Lang
     const {
       flows,
       resources,
-    } = this
-    return JSON.stringify(
-      convertKeysCase({
-        flows,
-        resources,
-      },
-      'SNAKE',
-      ['platformMetadata', 'ioViamo']),
-      null,
-      2,
-    )
+    } = this as { flows: IFlow[]; resources: IResourceDefinition[] }
+    return {
```

Comment:
Can we just return directly?

Reply:
Apparently I forgot to `JSON.stringify`, corrected it, and at the end I'd prefer to leave the destruct approach